bubuku/controller.py

```diff
@@ -133,6 +135,11 @@ def _run_changes(self, running_changes: dict) -> list:
                             changes_to_remove.append(change.get_name())
                         else:
                             _LOG.info('Action {} will be executed on next loop step'.format(change))
+                            time_till_next_run = change.time_till_next_run()
+                            if min_time_till_next_change_run is None:
+                                min_time_till_next_change_run = time_till_next_run
+                            else:
```

Comment:
```suggestion
                            elif time_till_next_run is not None:
```

Reply:
but it's nitpicking feel free to merge as it is.

Reply:
What is semantics of `None`?  The method is not even supposed to return that (not optional).